projects/angular/src/wizard/wizard.html

```diff
@@ -14,15 +14,16 @@
   [clrModalLabelledById]="wizardId"
 >
   <nav class="modal-nav clr-wizard-stepnav-wrapper">
-    <h1 class="clr-wizard-title" id="{{wizardId}}"><ng-content select="clr-wizard-title"></ng-content></h1>
+    <h1 *ngIf="title" class="clr-wizard-title" id="{{wizardId}}">{{title}}</h1>
+    <ng-content select="clr-wizard-title"></ng-content>
```

Comment:
I believe this is a breaking change. If a consumer doesn't change their application code, the heading element will be lost.